src/components/Navigation.vue

```diff
@@ -235,5 +245,10 @@ to {
 .v-popper__inner {
 	height: unset !important;
 }
+.error-message {
```

Comment:
the usual remark

Reply:
i was not sure if it was the BEM, or the space above, so i did both